swift/Samples/SampleApp/Podfile.lock

```diff
@@ -8,10 +8,16 @@ PODS:
   - Workflow/Tests (0.9.0):
     - ReactiveSwift (~> 5.0.0)
     - Result
+  - WorkflowUI (0.9.0):
```

Comment:
Next time we do a version bump, do we need to change all these numbers too?

Reply:
It isn't strictly necessary, but if they are not updated subsequent `bundle exec pod install` calls will leave a dirty `Podfile.lock`